Homework7/Task3/Polaroid.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Homework7.Task3
+{
+    public class Polaroid : Device, IPrint, IPhoto
+    {
+        public int PaperWidth { get; set; }
+        public int PaperHeight { get; set; }
+
+        public double NumberOfPixelsInCamera { get; set; }
+
+        public new string Description { get => $"Price: {Price}, model:{ModelName}, number of pixels in camera: {NumberOfPixelsInCamera}"; }
```

Comment:
You have common part in Device class. Please how it can be improved to avoid duplicate code. The same in MobilePhone class

Reply:
Fixed